test/test_marshal.lua

```diff
@@ -0,0 +1,103 @@
+local test = require 'regress'
+local ipc = require 'libipc'
+
```

Comment:
Maybe add a test that provides the sizes too? Just to make sure it doesn't crash.